ckanext/validation/plugin.py

```diff
@@ -188,6 +186,10 @@ def after_create(self, context, data_dict):
             for resource in data_dict.get(u'resources', []):
                 self._handle_validation_for_resource(context, resource)
         else:
+            for plugin in p.PluginImplementations(IDataValidation):
+                if not plugin.can_validate(context, data_dict):
+                    log.debug('Skipping validation for resource {}'.format(data_dict['id']))
+                    return
```

Comment:
It's a code dup from: `_handle_validation_for_resource`
We should either run here `self._handle_validation_for_resource` and decorate it if needed or create a new `_handle_special_validation_for_resource` which would share some private code with `_handle_validation_for_resource`

This solution has a lot of technical dept. We might be too busy atm however to handle it right now.